### PR TITLE
Apparel individual page

### PR DESCRIPTION
**Pull request by vivientam**
I have added images, build the contact and ourstory pages. Pls check.

### CODE REVIEW

File: app/assets/javascripts/apparels.js.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (h8rry) [inline app/assets/javascripts/apparels.js.coffee:1]: You can remove all the empty `.js.coffee` files

File: app/assets/javascripts/ourstories.js.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (h8rry) [inline app/assets/javascripts/ourstories.js.coffee:1]: You can remove all the empty `.js.coffee` files

File: app/assets/stylesheets/apparels.css.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the apparels controller here.
```
Comment (h8rry) [inline app/assets/stylesheets/apparels.css.scss:1]: You can remove all the empty `.js.coffee` files

File: app/assets/stylesheets/ourstories.css.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the ourstories controller here.
```
Comment (h8rry) [inline app/assets/stylesheets/ourstories.css.scss:1]: You can remove all the empty `.css.scss` files

File: app/controllers/collections_controller.rb [hunk 5]
```diff
@@ -2,6 +2,7 @@ class CollectionsController < ApplicationController
 
   def index
     @collections = Collection.all
+
```
Comment (h8rry) [inline app/controllers/collections_controller.rb]: no need for a line break

File: app/helpers/ourstories_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module OurstoriesHelper
```
Comment (h8rry) [inline app/helpers/ourstories_helper.rb]: remove empty files

File: app/helpers/apparels_helper.rb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+module ApparelsHelper
```
Comment (h8rry) [inline app/helpers/apparels_helper.rb]: remove empty files

File: app/views/apparels/show.html.erb [hunk 8]
```diff
@@ -0,0 +1,42 @@
+<div class="row">
+<h3 class="col-md-8 col-md-offset-2 col-xs-12 bold-text">2015 <%= @collection.name %> Collection</h3>
```
Comment (h8rry) [inline app/views/apparels/show.html.erb]: indentation

File: app/views/collections/index.html.erb [hunk 9]
```diff
@@ -16,6 +16,12 @@
           <div class="col-xs-9 collection-location">
           <%= collection.location %>
           </div>
+          <div>
+            <% collection.apparels.take(5).each do |apparel| %>
+              <%= image_tag("apparels/#{apparel.id}.jpg") %>
+            <% end %>
+          </div>
+
```
Comment (h8rry) [inline app/views/collections/index.html.erb:23]: don't need line break

**Comment by vivientam** [comment]
I may need those css.scss file to amend the style of the page. :)

**Pull request merged**